modules/n1ql/pages/advanced.adoc

```diff
@@ -27,8 +28,9 @@ The cost-based optimizer takes into account the cost of memory, CPU, network tra
 
 * xref:n1ql:n1ql-language-reference/cost-based-optimizer.adoc[Cost-Based Optimizer]
 
-== N1QL Support for Couchbase Transactions
+== {sqlpp} Support for Couchbase Transactions
 
-N1QL offers full support for Couchbase ACID transactions.
+{sqlpp} offers full support for Couchbase ACID transactions.
 
-* xref:n1ql:n1ql-language-reference/transactions.adoc[N1QL Support for Couchbase Transactions]
\ No newline at end of file
+* xref:n1ql:n1ql-language-reference/transactions.adoc[]
+* xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[]
```

Comment:
Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)